rotkehlchen/globaldb/upgrades/v4_v5.py

```diff
@@ -87,6 +87,20 @@ def _reset_curve_cache(write_cursor: 'DBCursor') -> None:
     )
 
 
+def _add_contract_abis(write_cursor: 'DBCursor', root_dir: Path) -> None:
+    """
+    Adds the following contract abis to the globaldb:
+    - Curve Metaregistry
+    """
+    with open(root_dir / 'data' / 'eth_abi.json') as f:
```

Comment:
No. Don't add it there. Just add it to the upgrade directly. That file is no longer used and was used only in an old DB upgrade.